### DISCUSSION TITLE
How do you change data directory location in Windows 11 ?

### DISCUSSION THREAD

**Discussion by tanreeve** [TriliumNext/Trilium#3466] [Q&A] [answered]
Hello, How do I change data directory location in Windows 11 ? 
I already read [wiki ](https://github.com/zadam/trilium/wiki/Data-directory) about data directory, but I can't find how to change data directory in Windows. 

Thank you.

**Comment by contributor** [comment, marked as answer]
On Windows 10 it works like this
trilium.bat
```
SET TRILIUM_DATA_DIR=%USERPROFILE%\AppData\Roaming\trilium-data
start %USERPROFILE%\Programs\Trilium\trilium.exe
```
I guess it is the same on Windows 11

**Reply by tanreeve** [reply]
should I run that bat file every time I open trilium or should I just run it once ? Sorry I'm new. 
Thank you

**Reply by contributor** [reply]
You use bat file to start trilium every time (instead of your current method of starting the app)
Additionally, you can make shortcut (lnk) file for the bat file and put it to Start Menu for example.

**Reply by tanreeve** [reply]
Oh ok I get it. Thank you so much.
